chef/cookbooks/hyperv/recipes/do_nova.rb

```diff
@@ -2,6 +2,7 @@
 
 include_recipe "hyperv::openstack_install_neutron"
 include_recipe "hyperv::openstack_install_nova"
+include_recipe "hyperv::openstack_install_networking-hyperv"
```

Comment:
i used a dash only because the package name has a dash http://tarballs.openstack.org/networking-hyperv/ but i will change it to underscore